src/Sulu/Component/Util/SuluVersionPass.php

```diff
@@ -15,31 +15,68 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\Finder\SplFileInfo;
 
+/**
+ * Read versions from composer.lock and composer.json.
+ */
 class SuluVersionPass implements CompilerPassInterface
 {
     /**
      * {@inheritdoc}
      */
     public function process(ContainerBuilder $container)
     {
-        $container->setParameter(
-            'sulu.version',
-            $this->getVersionFromComposerJson(realpath($container->getParameter('kernel.root_dir') . '/..'))
-        );
+        $dir = dirname(realpath($container->getParameter('kernel.root_dir')));
+
+        $container->setParameter('sulu.version', $this->getSuluVersion($dir));
+        $container->setParameter('app.version', $this->getAppVersion($dir));
     }
 
-    private function getVersionFromComposerJson($dir)
+    /**
+     * Read composer.lock file and return version of sulu.
+     *
+     * @param string $dir
+     *
+     * @return string
+     */
+    private function getSuluVersion($dir)
     {
         $version = '_._._';
 
         /** @var SplFileInfo $composerFile */
-        $composerFile = new SplFileInfo($dir . '/composer.json', '', '');
-        if ($composerFile->isFile()) {
-            $composerJson = json_decode($composerFile->getContents());
+        $composerFile = new SplFileInfo($dir . '/composer.lock', '', '');
+        if (!$composerFile->isFile()) {
+            return $version;
+        }
 
-            return $composerJson->version;
+        $composer = json_decode($composerFile->getContents(), true);
+        foreach ($composer['packages'] as $package) {
+            if ($package['name'] === 'sulu/sulu') {
+                return $package['version'];
+            }
         }
 
         return $version;
     }
+
+    /**
+     * Read composer.json file and return version of app.
+     *
+     * @param string $dir
+     *
+     * @return string
+     */
+    private function getAppVersion($dir)
+    {
+        $version = '_._._';
+
+        /** @var SplFileInfo $composerFile */
+        $composerFile = new SplFileInfo($dir . '/composer.json', '', '');
+        if (!$composerFile->isFile()) {
+            return $version;
+        }
+
+        $composerJson = json_decode($composerFile->getContents(), true);
+
+        return $composerJson['version'];
```

Comment:
This errs if no version is given in the `composer.json` file. I would assume that should be possible, since most simple project probably don't have a separate versioning.

I would also suggest that we remove the `version` attribute from sulu-standard and sulu-minimal.